service/package/rubygem-agama.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Mon Aug 21 11:15:50 UTC 2023 - Imobach Gonzalez Sosa <igonzalezsosa@suse.com>
+
+- Set the manager service as busy during the startup phase when
+  a single product is available (bsc#1213194).
```

Comment:
I would say that this change is misleading..it sets busy always and that fixed issue with single product. But set it **always** during startup